utils/src/activityLog/activities/ExecuteActivity.ts

```diff
@@ -36,6 +36,7 @@ export class ExecuteActivity<C extends types.ExecuteActivityContext> extends Act
                     id: nonNullProp(activityResult, 'id'),
                     name: nonNullProp(activityResult, 'name'),
                     type: nonNullProp(activityResult, 'type'),
+                    azExtResourceType: nonNullProp(activityResult, 'azExtResourceType'),
```

Comment:
If we want to merge this into main now without making breaking changes, we shouldn't require this here yet. 

I have a refactor of the Activity API almost ready, and we can include this change with those.

Reply:
the `AppResource` on `ActivityResult` gets passed to the azureResourceGroups.revealResource.

We could make azureResourceGroups.revealResource also accept a resourceId string, then we wouldn't have to require the full `AppResource` type on the `ActivityResult`.

Reply:
I think I like that better. The `AppResource` objects are the "property" of the host extension, it doesn't make a lot of sense that the client extension (via this activity feature) creates an `AppResource` to hand back to the RG extension.